maquetado/scss/application.scss

```diff
@@ -0,0 +1,34 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+body {
+	font-family: 'Roboto', sans-serif;
+	margin: 0;
+	padding: 0;
+}
+
+.nav {
+
+}
+
+.nav {
+	height: 90px;
+	width: 100%;
+	position: fixed;
+	box-shadow: 0px 5px 10px $grey;
+
+	.nv-logo {
+		width: 15%;
```

Comment:
try not to use % width to the logo as it will become really small when the widow size is small. Use a fix width